gen/calc.py

```diff
@@ -205,7 +205,7 @@ def validate_json_list(json_list):
 
         assert type(list_data) is list, "Must be a JSON list. Got a {}".format(type(list_data))
     except ValueError:
-        assert False, "Provided input was not valid JSON: "+json_list
+        assert False, "Provided input was not valid JSON: " + json_list
```

Comment:
Definitely aiming for that.  Many, [including this one I](https://github.com/dcos/dcos/pull/628/files#diff-3cd4ddcecc89d0887dd9cb3d5fabe995L208) removed in #628 as part of other refactoring / cleanup